src/libbpf/bpf.h

```diff
@@ -112,6 +112,11 @@ enum bpf_attach_type {
 	BPF_XDP_CPUMAP,
 	BPF_SK_LOOKUP,
 	BPF_XDP,
+	BPF_SK_SKB_VERDICT,
+	BPF_SK_REUSEPORT_SELECT,
+	BPF_SK_REUSEPORT_SELECT_OR_MIGRATE,
+	BPF_PERF_EVENT,
+	BPF_TRACE_KPROBE_MULTI,
```

Comment:
Mind syncing up the rest of the file too? So file doesn't get skewed too much.

Reply:
sure, will add the rest, thanks